jfrog_platform_self-hosted/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - jfrog_platform
-All changes to the JFrog platform integration will be documented in this file.
+All changes to the JFrog Platform (Self-hosted) integration will be documented in this file.
```

Comment:
Non-blocking nit: This comment was probably added a while back. We actually now show the CHANGELOG directly in the in-app tile! So this sentence can probably be removed since it doesn't make sense in all contexts where this data is shown.